### ISSUE TITLE
Disk checker kicks in even if requirements are not specified

### ISSUE THREAD

**Issue by a-palchikov** [gravitational/gravity#1748]
Reported from a 7.0.9 installation:
```
[  129.608996] cloud-init[1120]: [×] disk capacity at /proc is 0 (failed to validate storage requirements)
```

**Comment by burdzwastaken** [comment]
we are also seeing this behaviour in `7.0.9`:
```
[  118.100276] cloud-init[1123]: [31m[ERROR]: The following pre-flight checks failed:
[  118.104108] cloud-init[1123]: [×] disk capacity at /proc is 0 (failed to validate storage requirements)
```

**Comment by r0mant** [comment]
Released in [7.0.11](https://gravitational.com/gravity/docs/changelog/#7011-june-24th-2020), ports are tracked in https://github.com/gravitational/gravity/issues/1740.
